en/ability.json

```diff
@@ -305,103 +305,103 @@
   },
   "tangledFeet": {
     "name": "Tangled Feet",
-    "description": "Raises evasiveness if the Pokémon is confused."
+    "description": "Moves targetting the Pokémon have their accuracy halved while the Pokémon is confused."
   },
   "motorDrive": {
     "name": "Motor Drive",
-    "description": "Boosts its Speed stat if hit by an Electric-type move instead of taking damage."
+    "description": "The Pokémon takes no damage when hit by Electric-type moves. Instead, its Speed stat is boosted."
   },
   "rivalry": {
     "name": "Rivalry",
-    "description": "Becomes competitive and deals more damage to Pokémon of the same gender, but deals less to Pokémon of the opposite gender."
+    "description": "The Pokémon’s competitive spirit makes it deal 25% more damage to Pokémon of the same gender, but 25% less damage to Pokémon of the opposite gender."
   },
   "steadfast": {
     "name": "Steadfast",
-    "description": "The Pokémon’s determination boosts the Speed stat each time the Pokémon flinches."
+    "description": "The Pokémon’s determination boosts its Speed stat every time it flinches."
   },
   "snowCloak": {
     "name": "Snow Cloak",
-    "description": "Boosts the Pokémon’s evasiveness in snow."
+    "description": "Boosts the Pokémon’s evasiveness in snow or hail by 20%. It also takes no damage from hail."
   },
   "gluttony": {
     "name": "Gluttony",
-    "description": "Makes the Pokémon eat a held Berry when its HP drops to half or less, which is sooner than usual."
+    "description": "If the Pokémon is holding a Berry to be eaten when its HP is 1/4 or less such as a Starf Berry, it will instead eat the Berry when its HP drops to 1/2 or less."
```

Comment:
```suggestion
    "description": "If the Pokémon is holding a Berry to be eaten when its HP is 1/4 or less such as a Starf Berry, it will instead eat the Berry when its HP drops to half or less."
```